lighthouse-core/report/html/renderer/psi.js

```diff
@@ -57,7 +65,8 @@ function prepareLabData(LHResultJsonString, document) {
   scoreGaugeWrapperEl.removeAttribute('href');
 
   const finalScreenshotDataUri = _getFinalScreenshot(perfCategory);
-  return {scoreGaugeEl, perfCategoryEl, finalScreenshotDataUri};
+  const psiDescription = lhResult.i18n.rendererFormattedStrings.psiDescription;
```

Comment:
instead, let's just provide this in the right spot so they dont have to. 

here, `el` is the `perfCategoryEl`:
![image](https://user-images.githubusercontent.com/39191/45065976-ecc59e80-b070-11e8-8401-a1522e365958.png)


i was writing this but maybe you want to do it differently:

```js
const descriptionText = lhResult.i18n.rendererFormattedStrings.psiDescription;
const descriptionEl = dom.convertMarkdownLinkSnippets(descriptionText);
const catDescriptionEl = dom.find('.lh-category-header__description', perfCategoryEl);
catDescriptionEl.appendChild(descriptionEl);
```